### DISCUSSION TITLE
Support for SAML Authentication

### DISCUSSION THREAD

**Discussion by maswatha** [PlusAuth/oidc-client-js#15] [General] [answered]
Hello, This javascript library is good for OIDC authentication. Does this library works for OAuth2 SAML Authentication? . Thanks

**Comment by ayZagen** [comment, marked as answer]
Hi @maswatha,

OIDC and SAML are completely different protocols. But, there are some Identity Providers supports both of them. So, if your provider supports initiating SAML request with OIDC/OAuth2 than this library would work too.

Note: OIDC is built on top of OAuth2.

**Reply by maswatha** [reply]
Hi @ayZagen ,

Thank you for previous reply. I came across one of the issue i am currently facing for token refresh and mentioned in issue tab. Were you able check on it and any updates on that when it will be resolved. Your response will be very much appreciated. Thanks
